sceptre/config/reader.py

```diff
@@ -431,12 +432,15 @@ def _render(self, directory_path, basename, stack_group_config):
             j2_environment = Environment(**j2_environment_config)
             template = j2_environment.get_template(basename)
             self.templating_vars.update(stack_group_config)
-            rendered_template = template.render(
-                self.templating_vars,
-                command_path=self.context.command_path.split(path.sep),
-                environment_variable=environ
-            )
-
+            try:
+                rendered_template = template.render(
+                    self.templating_vars,
+                    command_path=self.context.command_path.split(path.sep),
+                    environment_variable=environ
+                )
+            except Exception as err:
+                raise SceptreException(
+                    f'{Path(directory_path, basename).as_posix()} - {type(err).__name__}({err})')
```

Comment:
Rather than `{type(err).__name__}({err})`, you should use `traceback.format_exception_only`. That's the proper machinery to use when formatting exceptions like that.

Reply:
Done